buddyboss/buddyboss-platform
# PROD-1439 #311 - Fix include param for platform document API

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform-api/issues/311.

### How to test the changes in this Pull Request:

1. Pull the changes
2.  Check endpoint - `site-url/wp-json/buddyboss/v1/document?include=10`  ( e.g- document id as 10 )
3. See the response

### Proof Screenshots or Video
https://www.awesomescreenshot.com/video/5920582?key=aca06b75b795e13ec5bb84f5f86e5423


### Other information:




### Changelog entry

> Fixed the include param for platform document API
